docs/source/usage.rst

```diff
@@ -773,6 +773,31 @@ as DictConfig, allowing attribute style access on the resulting node.
     >>> show(container["structured_config"])
     type: DictConfig, value: {'port': 80, 'host': 'localhost'}
 
+OmegaConf.resolve
+^^^^^^^^^^^^^^^^^
+.. code-block:: python
+
+    def resolve(cfg: Container) -> None:
+        """
+        Resolves all interpolations in the given config object in-place.
+        :param cfg: An OmegaConf container (DictConfig, ListConfig)
+                    Raises a ValueError if the input object is not an OmegaConf container.
+        """
+
+Normally interpolations are resolved on access time (lazily). 
```

Comment:
I would say either "on access" or "at access time", "on access time" sounds a bit weird -- to me at least

Reply:
yeah, re-worded before I saw this.